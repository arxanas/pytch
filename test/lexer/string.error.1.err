EXPECTED_END_OF_STRING[1020] in test/lexer/string.error.1.pytch, line 1, character 5:
Error: I was expecting a ' to end this string.
   +------------------------------------------------------+
   | test/lexer/string.error.1.pytch                      |
 1 | 'foo"                                                |
   |     ^ Error: I was expecting a ' to end this string. |
   | ^ Note: This is the beginning of the string.         |
 2 | "bar'                                                |
   +------------------------------------------------------+
EXPECTED_END_OF_STRING[1020] in test/lexer/string.error.1.pytch, line 2, character 5:
Error: I was expecting a " to end this string.
   +------------------------------------------------------+
   | test/lexer/string.error.1.pytch                      |
 1 | 'foo"                                                |
 2 | "bar'                                                |
   |     ^ Error: I was expecting a " to end this string. |
   | ^ Note: This is the beginning of the string.         |
   +------------------------------------------------------+
