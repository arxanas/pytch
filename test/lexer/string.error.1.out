string literal '\'foo"'
trailing newline '\n'
the end of a statement
string literal '"bar\''
trailing newline '\n'
the end of the file
